README.md

```diff
@@ -29,6 +29,54 @@
 5. To create the initially empty database tables, run `python manage.py
    migrate` inside the ncvoter folder where manage.py is.
 
+### External Database Drive Tips
+
+Are you running the project with a database sitting on an external drive? Here
+are some tips for workflows that make this process easier.
+
+#### Creating an external database
+
+You can move only one database on your postgres instance to a different drive,
+but you can't stop and start single databases. This means simply moving the TABLESPACE
+would still leave you with problems when disconnecting the external drive.
+
+Instead, create a whole separate cluster and run a separate postgres instance *only*
+when working on this project.
+
+    pg_ctl -D /Volumes/Untitled/postgres initdb
```

Comment:
so `pg_ctl` is not on the PATH by default for Ubuntu, [by design](https://askubuntu.com/a/385970/7131) (sigh)

Could we just add a note, something like, 'For Ubuntu, all of the postgresql binaries are located in /usr/lib/postgresql/9.6/bin, so you'll need to type out the full path or add that directory to your PATH environment variable'?